jax/_src/scipy/qdwh.py

```diff
@@ -0,0 +1,235 @@
+"""
```

Comment:
I'd be tempted to move this into `jax/_src/lax/qdwh.py`. This is mostly because we want to be able to call it from `jax.lax.linalg` primitives like SVD. Optional, though, I can do that in another PR.

Reply:
Moved